secure-releases.md

```diff
@@ -0,0 +1,60 @@
+# Secure Releases and CVE Management for the JS Ecosystem
+
+## Secure Releases
+
+In order for your JS software project to be safe, secure, and reliable, the versions it releases need to follow security best practices. Below are the OpenJS Foundations recommended best practices for secure releases.
+
+### Two-factor Authentication
+
+Two-factor authentication (2FA) means having something more than just your password to confirm you're you. The second factor can be a one-time code using an app like Google Authenticator, a hardware key (such as Yubikey or Titan), or a passkey using touch ID, face ID, or something else that's stored locally on your device. Note: do not *ever* use SMS for 2FA unless it's the only mechanism offered, since SMS is vulnerable to SIM swap and number forgery attacks.
+
+Although this extra authentication step can be a mild inconvenience, the benefit for security is overwhelmingly worth it. Both Github and npm will soon require 2FA for every user, so you might as well enable it now.
```

Comment:
there's very little left in the year. I'd suggest just changing it now to say the GH already requires it

Reply:
They don't quite require it yet, though; I'm happy to update the doc after that's changed.